paper.tex

```diff
@@ -724,77 +724,71 @@ \section*{Genome ARGs}
 
 
 \section*{Ancestry resolution}\label{Ancestry_resolution}
-% This is a rough first pass. Will clarify terminology later.
 % Need to introduce Hudson's algorithm as concerned with simulating the process of
 % the coalescent with recombination as efficiently as possible, and properly merge in
 % with this section.
 
-The definition of an ARG as a data structure in terms of
-individuals' monoploid genomes
-and their direct patterns of inheritance allows us to describe arbitrarily
-complex relatedness structures, free from the limitations imposed by
-coalescent approximations.
-An ARG is an intrinsically retrospective
-structure, defined with respect to the ancestry of
-a set of sampled genomes.
-The graph structure (the nodes and their connectivity through edges)
-captures a great deal of information about this ancestry,
-but on its own it cannot tell us about the precise relationships between
-the samples along the genome: do to this, we must construct the
-local (``marginal'') trees.
-
-We can define a sample-resolved gARG as a tuple $(S, E)$
-where $S \subset \mathbb{N}$ is a set of sample nodes and
-$E$ is a set of edges. Each element of $E$
-is a tuple $(c, p, I)$ such that $c, p \in \mathbb{N}$,
-$c$ is the child node, $p$ is the parent node and $I$ is the set of
-disjoint genomic intervals $(\ell, r]$
-over which genome $c$ inherits from $p$, and there is a path from
-$c$ to some non-empty subset of $S$ through the local trees.
-% Should probably define how we build the local trees somewhere.
-
+The edge annotations we have described in Fig 1 and 2 show the regions of genome
+that have been passed from a parent to its child in their entirety. In effect, all edges
+span the entire length of the genome, apart from those that connect two parents
+to a child in which they recombine. In this form of edge annotation, which we call
+``entire edge encoding'', it is inefficient to extract the local trees from the ARG structure.
```

Comment:
This phrase doesn't do anything for me - I think it'll just confuse things more. What's "entire" about it? I think "local" or "immediate" inheritance is much clearer.

Reply:
Sure, let's pick something that's the least confusing. We wanted to avoid "local" though. We can switch back to immediate (which I think is what we had before).

Reply:
As discussed online, changing to "immediate edge annotation(s)", with a comment about better names